keyboards/mt84/config.h

```diff
@@ -0,0 +1,105 @@
+ /* Copyright MT84RGB by MT <704340378@qq.com>
+  * 
+  * This program is free software: you can redistribute it and/or modify 
+  * it under the terms of the GNU General Public License as published by 
+  * the Free Software Foundation, either version 2 of the License, or 
+  * (at your option) any later version. 
+  * 
+  * This program is distributed in the hope that it will be useful, 
+  * but WITHOUT ANY WARRANTY; without even the implied warranty of 
+  * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the 
+  * GNU General Public License for more details. 
+  * 
+  * You should have received a copy of the GNU General Public License 
+  * along with this program.  If not, see <http://www.gnu.org/licenses/>. 
+  */
+#pragma once
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+##define DESCRIPTION     mt84
```

Comment:
```suggestion

```
But you'll need to add `VENDOR_ID`, `PRODUCT_ID` and others.
See the [template](https://github.com/qmk/qmk_firmware/blob/master/quantum/template/avr/config.h).